test/new-e2e/tests/security-agent-functional/security_agent_test.go

```diff
@@ -0,0 +1,98 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package secagentfunctional
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments"
+	awshost "github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments/aws/host"
+	"github.com/DataDog/datadog-agent/test/new-e2e/tests/windows"
+	windowsCommon "github.com/DataDog/datadog-agent/test/new-e2e/tests/windows/common"
+	windowsAgent "github.com/DataDog/datadog-agent/test/new-e2e/tests/windows/common/agent"
+	componentsos "github.com/DataDog/test-infra-definitions/components/os"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/ec2"
+	"github.com/stretchr/testify/require"
+)
+
+type vmSuite struct {
+	e2e.BaseSuite[environments.Host]
+
+	testspath string
+}
+
+var (
+	devMode = flag.Bool("devmode", false, "run tests in dev mode")
+)
+
+func TestVMSuite(t *testing.T) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
I would give a more specific name, helps readability in Test Visibility
```suggestion
func TestSeucrityAgentWindowsSuite(t *testing.T) {
```

Reply:
so, since this came up in another comment...
and for future reference... 

which of these names appear where?
Many of the e2e tests use `TestVMSuite()` as the main entry point.

And then I changed
`TestSystemProbeSuite()`
to
`TestSystemProbeCWSSuite` based on the other feedback.

Reply:
OK.  But, just for my understanding.
The `junit report` you linked is per-job, not per pipeline.  So, I am still wondering about this:

If the ci job only runs one suite; my understanding is that there is nothing to confuse it with.  Is that correct or no?

Reply:
Your understanding is correct. There is no collision in reports. 

The issue is for humans: if there is a failure reported from this test, it can help to quickly know which part of the agent it covers. Without explicit name you need to jump into code and look at it. It is preferable to avoid the need to jump into code, if it can be avoided.